### DISCUSSION TITLE
The convert command not working with PDF files

### DISCUSSION THREAD

**Discussion by zBernie** [ImageMagick/ImageMagick#3878] [General]
I've tried various permutations of the "convert" command below on Windows 10, which does not work.  No errors are returned it runs, then nothing.  I'm using IM 7.0.3 and Ghostscript 9.52.  I'm trying to rotate a .PDF file 90 degrees.

convert -density 300x300 originalfile.pdf -rotate 90 test.pdf

-TIA

**Comment by fmw42** [comment]
try

`magick -density 300x300 originalfile.pdf -rotate 90 test.pdf`

You would have had to install the "legacy" components in your Windows install of ImageMagick to get the IM 6 convert. In IM 7, it is now called magick.

**Reply by zBernie** [reply]
That works, thank you!
